src/data/index.js

```diff
@@ -1820,4 +1820,52 @@ export default [
       'Bloomberg',
     ],
   },
+  {
+    id: 164,
+    name: 'Longest Word in Dictionary',
+    url: 'https://leetcode.com/problems/longest-word-in-dictionary/',
+    pattern: 'Trie',
+    difficulty: 'Easy',
+    companies: [],
```

Comment:
The companies for `'Longest Word in Dictionary'` are: `'Google', 'Goldman Sachs'`

![image](https://user-images.githubusercontent.com/13009507/82843130-3344d780-9eaa-11ea-886c-36525454b14f.png)